models/pattern/stages/validate.go

```diff
@@ -28,7 +29,9 @@ func Validator(prov ServiceInfoProvider, act ServiceActionProvider) ChainStageFu
 				act.Terminate(fmt.Errorf("invalid workload of type: %s", svc.Type))
 				return
 			}
-
+			if k8s.Format {
+				k8s.Format.DePrettify(svc.Settings)
```

Comment:
Is this only applicable for Services?

Reply:
Or is this applied generally?

Reply:
I presume that "Service" in this case is Pattern "Service" not Kubernetes "Service", is that right?

Reply:
Yes to your last statement.

Reply:
This makes much sense. Thanks for confirming.